vite.config.js

```diff
@@ -82,6 +82,9 @@ export default defineConfig(({ mode }) => {
             // Default naming fallback
             return "assets/[name]-[hash][extname]";
           },
+          manualChunks: {
```

Comment:
This *should* be not necessary once https://github.com/matrix-org/matrix-rust-sdk-crypto-wasm/pull/167 lands, as it will ship the WASM binary separately

Reply:
and in the meantime we can use this 👌

Reply:
I added an inline comment so that we don't forget.